App/LoginView.js

```diff
@@ -0,0 +1,29 @@
+function LoginView(){
+
+}
+
+LoginView.prototype.showInvalidUsername = function(){
+  console.log("showInvalidUsername");
+  document.getElementById("username").style.backgroundColor = "rgba(200,0,0,0.3)";
```

Comment:
Lot of repetition. Solve it by:
1. get the DOM node once and put it in a var(also reads way better)
2. Define a class in CSS(pay  very good  attention to the names of the classes, make sure they're extremely well readable): eg login-error, login-success etc and just add a class here instead of defining inline styles